README.md

```diff
@@ -68,6 +69,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+I called it in the each loop. Dunno where else I'd put it.
```

Comment:
I should have asked "Should we call strip_punctuation inside or outside the analyze_mood method? Why?"